pages/index.js

```diff
@@ -71,19 +73,50 @@ export default function Home() {
     setTask(updatedTaskList);
   }
 
-  return (
-    <Wrap>
-      <WrapMask>
+  function appendTask(title, durationHour, durationMinute) {
+    const durationHourInput = durationHour === "0" ? "" : durationHour + "h";
+    const durationMinuteInput =
+      durationMinute === "0" ? "" : durationMinute + "min";
+    const newTasks = [
+      ...task,
+      {
+        id: generateID(),
+        taskName: title,
+        duration: durationHourInput + durationMinuteInput,
+        checked: false,
+      },
+    ];
+    setTask(newTasks);
+  }
+
+  function HomePage() {
+    return (
+      <>
         <Welcome />
         <DateStyle>{getCurrentDate()}</DateStyle>
         <TaskCompleted task={task} handleToggleTask={handleToggleTask} />
-        <Task task={task} handleToggleTask={handleToggleTask} />
+        <Task
+          task={task}
+          handleToggleTask={handleToggleTask}
+          setPage={setPage}
+        />
+      </>
+    );
+  }
+
+  return (
+    <Wrap>
+      <WrapMask>
+        {page === "home" && <HomePage />}
+        {page === "create" && (
+          <Create appendTask={appendTask} setPage={setPage} />
+        )}
       </WrapMask>
```

Comment:
Why don't you use the default way of implementing routing in next.js? If you struggle sharing the state between two sites, have a look into Angelas latest code or ask her.

Reply:
@F-Kirchhoff thanks for this suggestion! already finished!